modules/lambda-function/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "name" {
+  description = "The lambda function name."
```

Comment:
No default?

Reply:
I removed it. I guess I could use a default. I figured if someone is using only this module they have a more advanced use case, so I moved the default up one level to the parent module. Hate re-declaring defaults, heh.

Reply:
Assume that all modules MAY be used independently.

Reply:
Done